plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -81,7 +81,10 @@ class NewCreateServiceModalForm extends Component {
     );
 
     // Render initial app config
-    this.state.appConfig = this.getAppConfig(this.state);
+    this.state.appConfig = this.getAppConfig(
```

Comment:
Actually I realised that I am creating the `appConfig` almost every time I am creating the new state, so I moved the `appConfig = ...` inside the `getNewStateForJSON`